ops/cloud-deployment/ustp-cams-cosmos.bicep

```diff
@@ -18,10 +18,10 @@ param analyticsWorkspaceId string = ''
 param allowAllNetworks bool = false
 
 @description('Action Group Name for alerts')
-param actionGroupName string
+param actionGroupName string = ''
```

Comment:
@brandonjohnsen @davemedvitz @jsheehan6 this is what would fix needing to pass in those empty params when we run the bicep deploy on cosmos for the next run